cacher/cacher.go

```diff
@@ -131,9 +133,24 @@ func (cacher *Cacher) dispatch(id [32]byte, msg http.RequestWithResponder) {
 					return true
 				}
 
+				if tx.Tx.Selector.IsCrossChain() || tx.Tx.Selector.IsIntrinsic() {
```

Comment:
Don't we want to decode the output if it's a cross-chain transaction rather than returning here?

Reply:
Misinterpreted your previous comment; thought cross chain in this case means that the tx was host > host